advanced/list-of-log-names.md

```diff
@@ -70,4 +69,4 @@ This category is for debugging purposes. Logs in this category are meant to aid
 
 | Name                | Emitted when     |
 | ------------------- | ---------------- |
-| REACTION_ROLE_DEBUG | Someone tries to get a join only role or something else goes wrong in the process of adding a reaction role |
+| REACTION_ROLE_DEBUG | Something goes wrong in the process of adding a reaction role |
```

Comment:
```suggestion
| REACTION_ROLE_DEBUG | An error occurs in the process of triggering a reaction role |
```

This is correctly generalized. Now add the link to the tutorial on the words "reaction role"